third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -75,9 +78,9 @@ juLog() {
     class="default"
   fi
 
-  echo "name is: $name"
-  echo "class is: $class"
-
+  # echo "name is: $name"
```

Comment:
Yeah. The lib already prints the complete command. Printing name and class are not very useful since they are from user input. And it makes the output some how crowded.